go.mod

```diff
@@ -1,35 +1,42 @@
 module github.com/uptime-industries/compute-blade-agent
 
-go 1.23.0
-
-toolchain go1.24.0
+go 1.24.0
```

Comment:
This will be breaking with the tinygo version pinned for the release pipeline (https://github.com/uptime-industries/compute-blade-agent/blob/main/.github/workflows/release.yaml#L46)

Gonna test the smart-fan-unit firmware in the coming days with the change&bump the tinygo version, then this is ready2go

Reply:
should be fixed now